.isort.cfg

```diff
@@ -9,4 +9,4 @@ line_length=88
 known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
-default_section=THIRDPARTY
+known_third_party=psycopg2,setuptools
```

Comment:
Please make a rebase to get the new isort config.

Reply:
Current branch 13.0-mig-base_search_fuzzy is up to date.

Reply:
Please undo this change and let what is in main branch.